backend/src/main/java/edu/netcracker/backend/util/AuthorityUtils.java

```diff
@@ -2,25 +2,33 @@
 
```

Comment:
Change package 
from backend/src/main/java/edu/netcracker/backend/util/...
to     backend/src/main/java/edu/netcracker/backend/utils/...